.kubernetes/kubernetes_deployments.yaml

```diff
@@ -29,7 +29,7 @@ spec:
         ports:
         - containerPort: 80
         env:
-        - name: ConnectionStrings__default
+        - name: ConnectionString__default
```

Comment:
this is incorrect. `GetConnectionString` method looks in the `ConnectionStrings` section.

Reply:
ah, i see, you changed that. ok, as long as it's the same place the code looks